istio-telemetry/prometheus-operator/values.yaml

```diff
@@ -0,0 +1,18 @@
+# Controls the default scrape interval used in the ServiceMonitors
```

Comment:
Please add a top-level "prometheus:", for consistency and compat with 1.1.
Can you do a diff with the values from 1.1, and maybe add a doc to any setting that is new, and 
a commented out setting if it was deleted ? 
( we need to do this for all components )